docs/guides/device-gateway/index.md

```diff
@@ -9,10 +9,10 @@ ngrok allows you to create secure ingress to any app, IoT device, or service wit
 
 This section provides getting started guides for adding ngrok to the most popular IoT devices, ensuring the agent runs integrated to your operating system, restricting traffic to trusted origins, and integrating traffic events with your preferred logging tool.
 
-| Name                                                | Description                                                                                       |
-| --------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| [APIs on Devices](/guides/device-gateway/agent)     | Configure site-to-site connectivity to APIs on devices using the ngrok agent with mTLS encryption |
-| [Linux](/guides/device-gateway/linux)               | Configure ssh access to remotely manage Linux devices using ngrok                                 |
-| [Raspberry Pi](/guides/device-gateway/raspberry-pi) | Configure ssh access to remotely manage Raspberry Pi devices using ngrok                          |
-| [Raspberry Pi OS](/guides/device-gateway/raspbian)  | Configure ssh access to remotely manage Raspberry Pi OS (formerly Raspbian) devices using ngrok   |
-| [Windows](/guides/device-gateway/windows)           | Configure ssh access to remotely manage Windows devices using ngrok                               |
+| Name                                                                 | Description                                                                                     |
+| -------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| [APIs on Devices with the ngrok agent](/guides/device-gateway/agent) | Connect to APIs on devices using the ngrok agent with mTLS encryption                           |
+| [ Linux ](/guides/device-gateway/linux)                              | Configure ssh access to remotely manage Linux devices using ngrok                               |
```

Comment:
not sure if this was auto-added somehow but it might not render correctly with the spaces